packages/SwingSet/src/kernel/kernel.js

```diff
@@ -214,7 +213,7 @@ export default function buildKernel(
 
   const kernelSyscallHandler = makeKernelSyscallHandler({
     kernelKeeper,
-    storage: enhancedCrankBuffer,
+    kvStore: enhancedCrankBuffer,
```

Comment:
I suspect we'll be wanting to pass the whole `hostStorage` object in here soon, but with this refactoring in place, that ought to be pretty trivial.